book__a__table.css

```
@@ -1,4 +1,4 @@
-.book__a__table__block {
+.book__a__table-block {

```

аналогично комментарию выше

```suggestion
.book__a-table-block {

```

---

либо вот так

```css
.book-a__table-block { 
```

---

исправил - залил исправленное